StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedUpdateContext.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  EmbeddedUpdateContext.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 2/14/25.
+//
+
+import Foundation
+
+struct EmbeddedUpdateContext {
```

Comment:
This is pretty similar to PaymentSheetFlowController.UpdateContext, but embedded updates have the canceled case and FC doesn't. Appears FC updates can't be canceled as we ignore everything but the latest update.